libraries/dynamic-ui-react/src/style/abstracts/variables/_+import.scss

```
@@ -102,7 +102,6 @@ $input-btn-focus-width: 0 !default;
 $input-btn-focus-color-opacity: 1 !default;
 $input-btn-focus-color: rgba(var(--#{$prefix}secondary-rgb), $input-btn-focus-color-opacity) !default;
 $input-btn-focus-blur: 0 !default;
```

por acá dejemos la definición que propone bootstrap que es la que va a tomar por default 

```suggestion
$input-btn-focus-blur: 0 !default;
$input-btn-focus-box-shadow: $focus-ring-box-shadow !default;
```